rllib/env/tests/test_multi_agent_env.py

```diff
@@ -142,7 +142,8 @@ def policy_mapping_fn(agent_id, episode, worker, **kwargs):
             env_creator=lambda _: BasicMultiAgent(5),
             default_policy_class=MockPolicy,
             config=AlgorithmConfig()
-            .rollouts(rollout_fragment_length=50, num_rollout_workers=0)
+            .rollouts(rollout_fragment_length=50, num_rollout_workers=0,
+                enable_connectors=True)
```

Comment:
... also we should remove this here to see if test still passes with conenctors disabled